content/ar/courses/350/blog-project/models.md

```diff
@@ -1,117 +1,118 @@
 ---
-# Page title
-title: The Blog Project Models
+#عنوان الصفحة
+العنوان: نماذج مشروع المدونة
 
-# Title for the menu link if you wish to use a shorter link title, otherwise remove this option.
-linktitle: Models
 
-# Date page published
-date: 2021-03-23
+# عنوان رابط القائمة إذا كنت ترغب في استخدام عنوان ارتباط أقصر ، وإلا فقم بإزالة هذا الخيار .
+عنوان الارتباط: النماذج
 
-# Academic page type (do not modify).
-type: book
+# تاريخ نشر الصفحة
+التاريخ: 2021-03-23
 
-# Position of this page in the menu. Remove this option to sort alphabetically.
-weight: 2
+# نوع الصفحة الأكاديمية (لا تعدل).
+النوع: كتاب
 
-draft: False
+#مكان هذه الصفحة في القائمة. قم بإزالة هذا الخيار للترتيب أبجديًا.
+الوزن: 2
+
+مسودة: خطأ
```

Comment:
@Naeimah everything above the --- mark should not be translated. Please return it back to english.

the only 2 lines that can be translated are the title and linktitle and they should look like this:

```
title: نماذج مشروع المدون
linktitle: النماذج
```

because its data, please fix